spec/support/omniauth_macros.rb

```diff
@@ -0,0 +1,45 @@
+module OmniauthMacros
```

Comment:
This module seems a bit weird to me. First of all, this name, we don't use the word `macros` in the ruby world, `helper` is better. Secondary, I don't like the side effect. When I call `#github_auth_hash`, I am not waiting for initialization of `OmniAuth.config.mock_auth[:github]`. Finally, I don't see a reason to include it in each spec file. Looks like we need it only in `spec/services/omni_auth_service_spec.rb` so I propose to get rid of this file.

Reply:
In `spec/services/omni_auth_service_spec.rb`
```ruby
let(:omni_auth_hash) do
  OmniAuth::AuthHash.new({
    provider: 'provider',
    uid: '12345',
    info: { email: 'user@email.com' }
  })
end
```

Reply:
I'm new to this OmniAuth testing, so, used some guides from the Internet.